src/components/referrals/ReferralsGroupComponent.vue

```diff
@@ -1,6 +1,8 @@
 <template>
     <section class="referralsContainer">
-        <h3 class="referralsContainer__header">
+        <h3
+            class="referralsContainer__header"
+            tabindex="0">
```

Comment:
not sure about reading num of items same as beginning of page

Reply:
its reding good bro 👯